src/core.js

```diff
@@ -820,6 +823,7 @@ goog.require('olcs.core.OlLayerPrimitive');
    */
   olcs.core.olPointGeometryToCesium = function(geometry, projection, style,
       billboards, opt_newBillboardCallback) {
+    var altitudeMode = geometry.get('altitudeMode');
```

Comment:
Not sure if it's good to rely on a per-geometry object setting for this. The workflow for clamping all geometries of all features to the ground is a bit weird then from an API perspective. I think it would be better to make this an OLCesium instance option.